src/departements.js

```diff
@@ -0,0 +1,14 @@
+export const normalizeDpt = dpt => {
+  if (isNaN(dpt)) {
+    return dpt.toUpperCase(); // 2A, 2B
+  }
+  if (parseInt(dpt) < 10) {
+    return "0" + dpt;
+  }
+  return dpt;
```

Comment:
```suggestion
  const dtpNumber = Number(dpt)
  if (isNaN(dtpNumber)) {
    return dpt.toUpperCase(); // 2A, 2B
  }
  if (dtpNumber < 10) {
    return "0" + dpt;
  }
  return dtpNumber;
```

What you think @revolunet ?

Reply:
LGTM